CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### v0.4.6 (2021-04-19)
+
+- update usage of Buffer
+
```

Comment:
This file is autogenerated, so it's not necessary to update it as part of the PR. No worries, though, as it'll be overwritten when we make the next release :)

Reply:
cleanned.